chromium_src/sandbox/policy/features.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_SANDBOX_POLICY_FEATURES_H_
+#define BRAVE_CHROMIUM_SRC_SANDBOX_POLICY_FEATURES_H_
+
+#include "src/sandbox/policy/features.h"  // IWYU pragma: export
+
+namespace sandbox::policy::features {
+
+// Enables patching of executable's name from brave.exe to chrome.exe in
+// sandboxed processes.
+SANDBOX_POLICY_EXPORT BASE_DECLARE_FEATURE(kModuleFileNamePatch);
```

Comment:
[11576:31008:0712/214017.543:FATAL:feature_list.cc(654)] Check failed: CheckFeatureIdentity(feature). ModuleFileNamePatch has multiple definitions. Either it is defined more than once in code or (for component builds) the code is built into multiple components (shared libraries) without a corresponding export statement